server/app/repository/ApplicationRepository.java

```diff
@@ -128,14 +129,16 @@ private Application submitApplicationInternal(
           continue;
         }
         boolean isDuplicate = applicant.getApplicantData().isDuplicateOf(app.getApplicantData());
+        if (isDuplicate) {
+          throw new DuplicateApplicationException();
```

Comment:
I added a note to the exception class with more detail, but I'm not sure we should be using an exception here.

That said, I'm not immediately sure how to refactor this to avoid this strategy but happy to talk further or be convinced otherwise.

Reply:
@melanie-exygy can you say more about why you think a log would be helpful? I removed the log since this PR prevents saving dupes to the db. I thought it would be extra noise to have a log saying an applicant attempted to submit a duplicate.

Reply:
Since the exception is used for control flow and we are handling it with a screen for the user I'm not seeing much benefit of logging it. At most maybe just info level.

If I'm looking through my prod logs would there be anything I need to do or report about a warning? If not it creates extra noise and distraction from other potential issues.

Reply:
I looked at the existing logs and and an info log might be helpful, otherwise there's no indication that the applicant did not successfully submit:
![image](https://github.com/civiform/civiform/assets/24319951/d1b63ed8-98b7-4158-a105-92d3bc107db3)

There's no action to take from this, so I think info level is good. I'll add that.

Reply:
I was thinking that we might want to know if users keep trying to create duplicates so that we can address the UX root cause of their confusion.  It's probably unnecessary, though, as you and Gwen said, since we're preventing duplicates from happening.  Feel free to leave it out.

Reply:
Done! Added an info log. I had to update the `logback.xml` file to allow info level logs from `ApplicationRepository`. Lmk if you see any problems with doing that.